index.html

```diff
@@ -205,7 +205,7 @@ <h2>
         </p>
       </section>
       <section data-dfn-for='ScreenOrientation' data-link-for=
-      'ScreenOrientation'>
+      'ScreenOrientation'  id='widl-Screen-orientation'>
         <h2>
```

Comment:
Let’s remove this one for now. I’ll show you a different way to fix this (we want ReSpec to manage the IDs).